academy/xl-run-prod/4-network.md

```diff
@@ -7,16 +7,171 @@ tag: deep-dive
 
 # Prepare and connect to other nodes
 
-## Connection
+With the genesis created and received, a node operator needs to join the eventual network. In practice this means two things:
 
-Set up addresses, ports and keys to connect to other nodes.
+1. To open your node to connections from other.
+2. To know where are the other nodes, or at least a subset of them so that your node can attempt to connect to them.
 
-## Authentication
+In this section, you concern yourself with Tendermint and the peer-to-peer network. Other niceties like gRPC and REST into your Cosmos application are different concerns.
 
-Identify valid nodes.
+## Set up 
+
+As a node operator, from the time of genesis or at any time in the future, and on each machine, you first run an `init` command to at least set up the folders and pick an ASCII-only moniker:
+
+```sh
+$ ./myprojectd init lascaux
+```
+
+Overwrite the genesis created again with the real agreed one. While you are at it, you can make it read-only:
+
+```sh
+$ curl http://example.com/genesis.json -o ~/.myprojectd/config/genesis.json
+$ chmod a-wx ~/.myprojectd/config/genesis.json
+```
+
+The `init` command has also created a number of configuration files:
+
+```sh
+$ ls ~/.myprojectd/config
+```
+
+Should return:
+
+```txt
+addrbook.json
+app.toml
+client.toml
+config.toml <-- configuration for Tendermint
```

Comment:
Good idea to explain which file is used for what. Maybe we should add comments for all?